modules/ROOT/pages/configure-horizontal-autoscaling.adoc

```diff
@@ -0,0 +1,200 @@
+
+----
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: my-app
+  namespace: app-namespace
+spec:
+  behavior:
+    scaleDown:
+      policies:
+      - periodSeconds: 15
+        type: Percent
+        value: 100
+      selectPolicy: Max
+      stabilizationWindowSeconds: 300
+    scaleUp:
+      policies:
+      - periodSeconds: 180
+        type: Percent
+        value: 100
+      selectPolicy: Max
+      stabilizationWindowSeconds: 0
+  maxReplicas: 3
+  metrics:
+  - resource:
+      name: cpu
+      target:
+        averageUtilization: 70
+        type: Utilization
+    type: Resource
+  minReplicas: 1
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: my-app
+
+----
+
+Some points to consider:
+
+* Up to 100% running pods may be added until the maximum configured replicas is reached.
+* The number of pods added is based on the aggregated calculations over the past 180 seconds.
+* The number of pods removed is based on the aggregated calculations over the past 300 seconds.
+* Max scale up profile is 1 -> 2 -> 4 -> 8, where MuleSoft hits 8 replicas in approximately 6 minutes.
+
+The following are few examples of Mule app types that fit CPU based HPA:
+
+* Mule apps that scale based on CPU usage only.
+* HTTP/HTTPS apps with async requests.
+
+
+== Performance Considerations and Limitations
+
+In Runtime Fabric, the policy in use was benchmarked for Mule apps with CPU Reserved: 0.45vCpu and Limit: 0.55vCpu, which corresponds to these settings:
+
+----
+        resources:
+          limits:
+            cpu: 550m
+          requests:
+            cpu: 450m
+----
+=== Limitations
+Horizontal autoscaling does not work with clustering and rate limiting.
+
+
+== Configure Horizontal Autoscaling
+
+To configure horizontal autoscaling for Mule apps deployed to Runtime Fabric, follow these steps:
+
+. Enable the https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-metrics-apis[support for metrics APIs^] on your managed K8s API server.
+. From Anypoint Platform, select *Runtime Manager* > *Applications*.
+. Click *Deploy application*.
+. In the *Runtime* tab, check the *Enable Autoscaling* box.
+. Set the minimum and maximum *Replica Count* limits.
+. Click *Deploy Application*.
+
+image::rtf-autoscaling.png[Runtime Manager UI with Enable Autoscaling field selected]
+
+
+
+== Autoscaling Status and Logs
+
+When an autoscaling event occurs and your Mule app with horizontal autoscaling scales up, you can check the *Scaling* status in the Runtime Manager UI by clicking *View status* in your application’s details window:
+
+image::rtf-autoscaling-status.png[Runtime Manager UI with Mule app and Scaling status]
+
+For beta, MuleSoft does not support autoscaling logs for Runtime Fabric. You can track the scaled up replicas startup and the number of replicas of your Mule apps by running the following `kubectl` command in your terminal:
```

Comment:
It is not autoscaling logs, it is application logs through Anypoint Monitoring, but I would not mention that since we may fix it during the course of the beta. Can we change to:
To retrospectively look back at autoscaling events for your RTF applications, you can track startup of scaled up replicas and the number of scaled up replicas and of your Mule through the following command in your terminal: